simpleSequential/isIntegerListEmpty.py

```diff
@@ -3,3 +3,7 @@
 # Ex:
 #   isIntegerListEmpty([])        # return True
 #   isIntegerListEmpty([0, 1, 2]) # return False
+from typing import List
+
+def isIntegerListEmpty(lst: List[int]) -> bool:
+    return len(lst) == 0
```

Comment:
This operation is heavy. What is the input is a list of 1 billions element. Then when you run this line of code, it will take hours to know if this list is empty or not. Hours in cloud computing might cost you a lot.

Change this one to something more efficient.